src/chat/exceptions/websocket-exception.filter.ts

```diff
@@ -4,12 +4,15 @@ import { BaseWsExceptionFilter } from '@nestjs/websockets';
 @Catch()
 export class WebSocketExceptionFilter extends BaseWsExceptionFilter {
   catch(exception: any, host: ArgumentsHost) {
-    const args = host.getArgs();
-    // event ack callback
-    if ('function' === typeof args[args.length - 1]) {
-      const ACKCallback = args.pop();
-      ACKCallback({ error: exception.message, exception });
-      console.error(exception.message, exception);
+    const ctx = host.switchToWs();
+    const client = ctx.getClient();
+
+    if (exception.response) {
+      client.emit('error', exception.response);
+    } else {
+      client.emit('error', { exception });
     }
```

Comment:
```suggestion
    client.emit('error', exception.response || { exception });
```
이렇게도 처리할 수 있을거같아요 비교해보고 더 좋은 로직으로 수정되면 좋을듯 합니다.